README.md

```diff
@@ -46,21 +46,33 @@ features = ["libm"] # Makes it use libm instead of std for float math
+
+A *color space* which uses tristimulus values is often defined in terms of its *primaries* and *reference white* or *white point*. The *primaries* are each themselves a 3d vector of tristimulus values, and together represent the total *gamut* (i.e. displayable range of colors) of that color space. The *white point* defines what tristimulus values correspond to the color given off by a white reflector being lit by some known light source and being observed by the 'standard observer' (i.e. a standardized model of human color perception).
+
+The informal "RGB" color space is such a tristimulus color space, but it is underspecified since we don't know which primaries are being used (i.e. how exactly are the canonical "red", "green", and "blue" defined?). In most cases, when people talk about "RGB" or "Linear RGB" colors, what they are *actually* talking about is "Linear sRGB" color space, which uses the primaries and white point defined in the sRGB standard, but which *does not* have the non-linear transfer function applied.
+
+This library takes these things into account, and attempts to provide an interface which will let those who don't care so much about the intricacies of color still use colors correctly, while also allowing the advanced user a high degree of flexibility in how they use it.
 
 ## What It Can Do
 
 Palette provides tools for both color manipulation and conversion between color spaces. These are some highlights.
 
 ### Color Spaces
 
-RGB is probably the most widely known color space, but it's not the only one. You have probably used a color picker with a rainbow wheel and a brightness slider. That may have been an HSV or an HSL color picker, where the color is encoded as hue, saturation and brightness/lightness. There's also a group of color spaces that are designed to be perceptually uniform, meaning that the perceptual change is equal to the numerical change.
+"RGB" (which we now know is usually Linear sRGB) and other tristimulus based spaces like CIE Xyz are probably the most widely known color spaces, and is great when you want to perform physically correct math on color, like in a 2d or 3d rendering program, but there are also color spaces that are not defined in terms of tristimulus values.
```

Comment:
Just a small detail, but it may be good to make "(which we now know is usually Linear sRGB)" refer more clearly to the parts above, just to avoid confusion if someone TL;DRs it and skips to this part. :slightly_smiling_face: